services/uptime-kuma/service.yml

```
@@ -0,0 +1,27 @@
+version: 1
+
+id: uptime-kuma
+name: Uptime Kuma
+description: Uptime Monitoring Application
+color: '#5CDD8B' 
+icon: simpleicons/uptimekuma.svg
+
+environment:
+  - type: port
+    name: PORT
+    display_name: Port
+    default: 3001
+    description: The port to access uptime-kuma
+
+methods:
+  docker:
+    image: louislam/uptime-kuma
+    container_name: uptime-kuma
+    volumes:
+      uptime-kuma: /app/data
+    ports:
+      '3001/tcp': 3001
+    restart: always
+
+volumes:
+  uptime-kuma:
```

Here, the docker method description in Vertex doesn't follow exactly a `docker-compose.yml` file description. For example, there is no `container_name` tag in vertex as the name is already handled automatically by Vertex. Same for the `restart` and `volumes`!

```suggestion
methods:
  docker:
    image: louislam/uptime-kuma
    volumes:
      data: /app/data
    ports:
      '3001/tcp': 3001
```